tests/users/test_signup_view.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+
+from users.forms import SignUpForm
+
+
+class AccountsTestCase(TestCase):
+
+    def setUp(self):
+        self.user_data = {
+            'username': 'testuser124',
+            'first_name': 'test',
+            'last_name': 'demo',
+            'email': 'testuser@gmail.com',
+            'password1': '1234aditya_is_best',
+            'password2': '1234aditya_is_best'
+        }
+
+    def test_signup_page_elements_render(self):
+        """
+        Test for all the elements in the signup page is loding properly or not.
+        """
+        response = self.client.get(reverse('register'))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'users/register.html')
+        self.assertContains(response, '<input type="text" name="username" maxlength="150')
+        self.assertContains(response, '<input type="text" name="first_name"')
+        self.assertContains(response, '<input type="text" name="last_name"')
+        self.assertContains(response, '<input type="email" name="email"')
+        self.assertContains(response, '<input type="password" name="password1"')
+
+    def test_for_the_both_password_field_match(self):
+        """
+        Test tha password and confirm password field have same data or not.
+        """
+        response = self.client.post(reverse('register'), {
+            'username': 'testuser124',
+            'first_name': 'test',
+            'last_name': 'demo',
+            'email': 'testuser@gmail.com',
+            'password1': '1234adityaisbest11212',
+            'password2': '1234adityaisbest'
+        })
+        self.assertEqual(response.status_code, 200)
+        self.assertFormError(response, 'form', 'password2', "The two password fields didn't match.")
+
+    def test_for_form_valid(self):
+        """
+        Test for whole form is valid or not . By checking each field have have valid data or not
+        """
+        form = SignUpForm(data=self.user_data)
```

Comment:
Why is the form testing done in test_signup_view.py?

Move it to a separate file.